src/website/app/demos/Radio/examples/radio-group/importSyntax.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+export default {
+  id: 'import-syntax',
+  title: 'Import Syntax',
+  description: `\`\`\`
+import { RadioGroup } from 'mineral-ui/Radio';
+\`\`\`
+
+<Callout title="Note">
```

Comment:
shouldn't this be moved up into the header a la [Button](http://mineral-ui.com/components/button/) - "Always use the appropriate variant for your button, and write a clear, concise label to effectively communicate intention."

Reply:
I see this is in a callout in TextArea and TextInput as well but that could also be in the header there too. Or perhaps we should take that header text out of Button. Thoughts?

Reply:
Removed unrelated content from component (Radio, RadioGroup, TextInput, TextArea, Button) primary descriptions (aka header sections).